marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/helper/suite-setup.xqy

```diff
@@ -0,0 +1,42 @@
+xquery version "1.0-ml";
+import module namespace admin = "http://marklogic.com/xdmp/admin"
+  at "/MarkLogic/admin.xqy";
+
+(: add indexes for the test :)
+
+let $config := admin:get-configuration()
```

Comment:
If you'd like, you can add these to the test-config//final-database.json file so they don't have to be coded in here. I would change the entity name though so that it doesn't conflict with other tests that use a Customer entity - e.g. "MasteringTestEntity" or something like that.

Reply:
Done. Thanks for the suggestion.